src/server/api/area/index.ts

```diff
@@ -10,7 +10,7 @@ import { postCountry } from './postCountry'
 export const AreaApi = {
   init: (express: Express): void => {
     // Country
-    express.post(ApiEndPoint.Area.country(), AuthMiddleware.requireEditTableData, postCountry)
+    express.post(ApiEndPoint.Area.country(), AuthMiddleware.requireEditAssessmentStatus, postCountry)
```

Comment:
This end point should also be specific to edit assessment status.
It should be something like countrySstatus

Reply:
that endpoint allows to edit all the country properties, not only the status, so probably requireEditAssessmentStatus should be renamed into requireEditCountryAssessment

Reply:
canEditCountryProps and requireEditCountryProps ?
What do you think ?